procedures/katello/branch.md.erb

```diff
@@ -43,18 +59,22 @@
+- [ ] Create PR to [jenkins-jobs](https://github.com/theforeman/jenkins-jobs) to add a mapping between the current Foreman and Katello release branches. Also remove the oldest mapping from the config. See [this example](https://github.com/theforeman/jenkins-jobs/pull/54).
+- [ ] Review the Foreman schedule and planning ([example](https://community.theforeman.org/t/foreman-2-5-schedule-and-planning/22219)) and note the date of the first scheduled release candidate.
+- [ ] Generate and post the release procedure, if not already posted:
+  - [ ] (Note that Katello uses dots instead of dashes for release candidates, e.g. 4.1.1.rc1 not 4.1.1-rc1)
+  - [ ] Make sure to run `./tools.rb setup-environment` first
+  - [ ] Run `./tools.rb procedure release katello <%= release %>.rc1 <rc_release_date>` in [tool_belt](https://github.com/theforeman/tool_belt)
+  - [ ] Post the output in [Development](https://community.theforeman.org/c/development/9) with a "Releases" tag
 - [ ] Branch docs
-  * Use `./tools branch-docs config/katello/<%= release %>.yaml` to branch automatically or by hand:
-    - [ ] Copy [nightly manual content](https://github.com/theforeman/theforeman.org/tree/gh-pages/plugins/katello/nightly) to <%= release %>
-    - [ ] Create sidebar from [nightly](https://github.com/theforeman/theforeman.org/tree/gh-pages/_includes/plugins/katello/sidebar_nightly.html) to `sidebar_<%= release %>.html`
-    - [ ] Add <%= release %> to [layout](https://github.com/theforeman/theforeman.org/blob/gh-pages/_layouts/plugins/katello/documentation.html#L3) and set `display` to include `-RC`
-    - [ ] Add <%= release %> to [_config.yml]() mirroring other entries
   - [ ] Draft release notes in markdown ([example](https://gist.github.com/domcleal/5567450)), with these sections (and do not use personal pronouns):
     - [ ] Headline features: half a dozen important features with a few sentences description each
     - [ ] Upgrade notes: all important notices that users must be aware of before upgrading
-    - [ ] Release notes: bullet point list by category of most changes, excluding bug fixes for issues introduced during the release cycle, include link to bug numbers. You can auto-generate changes using the 'changelog' command in [tool_belt](https://github.com/theforeman/tool_belt)
-    - [ ] Add contributor list on release notes page ex `git shortlog -s KATELLO-3.17..KATELLO-3.18`
+    - [ ] Release notes: bullet point list by category of most changes, excluding bug fixes for issues introduced during the release cycle, include link to bug numbers. You can auto-generate changes using the `changelog` command in [tool_belt](https://github.com/theforeman/tool_belt) (make sure to run the `setup-environment` command first)
+    - [ ] Add contributor list on release notes page: `git shortlog -s KATELLO-4.0..KATELLO-4.1`
```

Comment:
Shouldn't this use variables instead? Something like:
```suggestion
    - [ ] Add contributor list on release notes page: `git shortlog -s KATELLO-4.0..KATELLO-<%= short_version %>`
```

It should be noted that the short version is now only available in release, but why is this part of the branching procedure? Normally that's a release procedure (because it should also be done for patch releases).

Reply:
@jjeffers @jturel What about moving this entire documentation section (Branch docs and API docs) to the release procedures?  You do need release notes before you can write a release announcement, but it seems to me they'd be more up to date if they were written after doing cherry-picks anyway.

Reply:
Is that compatible with having some form of release notes for the RC1 announcement? If so, that works for me but if we branch the documentation some time closer to the RC1 release and not the code branch date, don't we risk bringing in documentation into the release that's irrelevant(too new)?

Reply:
I suppose there's some risk, but given our track record with documentation I don't think it's too likely!